alts/src/main/java/io/grpc/alts/GoogleDefaultChannelCredentials.java

```diff
@@ -61,16 +66,26 @@ public static ChannelCredentials create() {
     return CompositeChannelCredentials.create(nettyCredentials, callCredentials);
   }
 
+  @SuppressWarnings("unchecked")
   private static InternalProtocolNegotiator.ClientFactory createClientFactory() {
     SslContext sslContext;
     try {
       sslContext = GrpcSslContexts.forClient().build();
     } catch (SSLException e) {
       throw new RuntimeException(e);
     }
+    Attributes.Key<String> clusterNameAttrKey = null;
+    try {
+      Class<?> klass = Class.forName("io.grpc.xds.XdsAttributes");
```

Comment:
This is an internal class without an internal prefix. The `io.grpc.xds` package is a mess currently. I don't think this name is likely to be stable. We need to clean it up, soon, for stuff like this.

Reply:
We are now at `io.grpc.xds.InternalXdsAttributes.ATTR_CLUSTER_NAME`.